pkg/proxy/apis/kubeproxyconfig/types.go

```diff
@@ -168,6 +177,13 @@ const (
 	ProxyModeKernelspace ProxyMode = "kernelspace"
 )
 
+type InterfaceType string
+
+const (
+	DefaultRoute  InterfaceType = "default-route"
+	AllInterfaces InterfaceType = "all-interfaces"
```

Comment:
This is redundant, isn't it?  If you want this, just leave the flag as "" ?

Reply:
I have the same feeling, let's remove this flag.

Reply:
Done.